javascript-if/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  }
+  return false;
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'j' || string[0] === 'J') {
+    return true;
+  }
+  return false;
+}
```

Comment:
To be fair, this should only return `true` if the string begins with a _capital_ `'J'`:

---

![image](https://user-images.githubusercontent.com/16143771/187777239-e74cbe70-34b3-4ec1-ab2f-81c86a850fdf.png)

Reply:
Definitely misread, thank you!